starport/templates/typed/new_stargate.go

```diff
@@ -74,13 +82,11 @@ func (t *typedStargate) protoRPCModify(opts *Options) genny.RunFn {
 		if err != nil {
 			return err
 		}
-		template := `%s
-	rpc All%s(QueryAll%sRequest) returns (QueryAll%sResponse);`
-		replacement := fmt.Sprintf(template, placeholder2,
-			strings.Title(opts.TypeName),
-			strings.Title(opts.TypeName),
-			strings.Title(opts.TypeName),
-		)
+		template := `%[1]v
+	rpc One%[2]v(QueryGet%[2]vRequest) returns (QueryGet%[2]vResponse);
```

Comment:
I would call it `Show[type]`

Reply:
applied this https://github.com/tendermint/starport/pull/417#issuecomment-727916380 in 4d69a84